ocp_install_env.sh

```diff
@@ -89,6 +114,18 @@ EOF
   fi
 }
 
+function dnsvip() {
+  local version
+
+  # dnsVIP was removed from 4.5
+  version=$(openshift_version | sed "s/\.//")
+  if [[ $version < 45 ]]; then
```

Comment:
technically `(( ))` compares numbers and `[[ ]]` compares strings, so the former appears to be what you were going for by removing the `.`?

Reply:
Ah yes you're right, or I guess we could test it with sort like in https://github.com/openshift-metal3/dev-scripts/blob/master/04_setup_ironic.sh#L176

Reply:
Updated to use sort instead